test/utils/describe-rule.js

```diff
@@ -0,0 +1,20 @@
+const getRulesData = require('./get-rules-data')
+const rules = getRulesData()
+
+/**
+ * 
+ * @param {String} groupName name of the `describe` block
+ * @param {Function} runTests function callback of `describe` block, which executes per rule
+ */
+const describeRule = (groupName, runTests) => {
+  rules.forEach(ruleData => {
+    const { filename } = ruleData
+    describe(filename, () => {
+      describe(groupName, () => {
```

Comment:
Ahh bummer. Looks like this nesting describe doesn't work in Jest. :( Might as well put this into a single `describe` then.

Reply:
@WilcoFiers It does work, see: https://circleci.com/gh/act-rules/act-rules.github.io/1276?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

![image](https://user-images.githubusercontent.com/20978252/57967934-400c2280-795c-11e9-9300-74f187883342.png)